App crashes when entering in the settings

### Steps to reproduce

1- Open the app
2- Go to settings
3- Open anything in settings
4- App crashes

### Expected behavior

The app works normally as usual

### Actual behavior

App crashes

### LibreTube version

Latest release from the source code

### Android version

13

### Other details

I build the release app using GitHub Actions in my fork, and this problem did not show up until recently, I built the debug apk and it works fine (except for the backup it does not export it at all).

I wonder if this will also show up in your next stable version.

I got this:

org.chromium.base.JniException: Unable to instantiate fragment com.github.libretube.ui.preferences.BackupRestoreSettings: make sure class name exists
	at androidx.fragment.app.FragmentManager$3.loadFragmentClass(SourceFile:30)
	at androidx.fragment.app.FragmentManager$3.instantiate(SourceFile:17)
	at androidx.preference.Preference.performClick(SourceFile:87)
	at androidx.preference.Preference$1.onClick(SourceFile:358)
	at android.view.View.performClick(View.java:7542)
	at android.view.View.performClickInternal(View.java:7519)
	at android.view.View.-$$Nest$mperformClickInternal(Unknown Source:0)
	at android.view.View$PerformClick.run(View.java:29476)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7940)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ExecInit.main(ExecInit.java:49)
	at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)
	at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:355)
Caused by: java.lang.ClassNotFoundException: com.github.libretube.ui.preferences.BackupRestoreSettings
	at java.lang.Class.classForName(Native Method)
	at java.lang.Class.forName(Class.java:454)
	at androidx.fragment.app.FragmentManager$3.loadClass(SourceFile:29)
	at androidx.fragment.app.FragmentManager$3.loadFragmentClass(SourceFile:3)
	... 17 more
Caused by: java.lang.ClassNotFoundException: com.github.libretube.ui.preferences.BackupRestoreSettings
	... 21 more
    

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

--- a/app/proguard-rules.pro
+++ b/app/proguard-rules.pro
@@ -110,3 +110,6 @@
 # Fix for miniplayer placing issue in release build
 -keep class androidx.constraintlayout.motion.widget.** { *; }
 -keepclassmembers class androidx.constraintlayout.motion.widget.** { *; }
+
+# Settings fragments are loaded through reflection
+-keep class com.github.libretube.ui.preferences.** { *; }

